app/mailers/payment_mailer.rb

```diff
@@ -7,10 +7,9 @@ def authorize_payment(payment)
     @payment = payment
     subject = I18n.t('spree.payment_mailer.authorize_payment.subject',
                      distributor: @payment.order.distributor.name)
+    to = payment.order.user ? payment.order.user.email : payment.order.email
     I18n.with_locale valid_locale(@payment.order.user) do
-      mail(to: payment.order.user.email,
-           from: from_address,
-           subject: subject)
+      mail(to: to, from: from_address, subject: subject)
```

Comment:
I'm wondering if there's a good reason to even try both of these values? Could we just go straight for `order.email` instead? We could drop line 10, and just do:

```diff
-- mail(to: to, from: from_address, subject: subject)
++ mail(to: payment.order.email, from: from_address, subject: subject)
```

What do you think?

Reply:
I thought about it, but didn't want to change to much the code, since I wasn't sure about exactly the consequences (are those two fields well synchronized?).
But for sure, I can take time to be sure that this is well relevant.

Reply:
Good point. I checked the AU production database:
```
Spree::Order.complete.where(email: [nil, ""]).count
=> 0

Spree::Order.joins(:user).complete.where("spree_orders.email != spree_users.email").count
=> 2561
Spree::Order.complete.count
=> 123145
# That's 2%, probably users changing their email at some point.
```

I think that we should simplify it. It should also satisfy Rubocop. :wink:

Reply:
Just to be sure to understand (I'd be happy to change this ;)): we will change the behavior for 2561 orders.

Thoughts:
 - maybe the user email changed between the time the order has been placed and now?
 - 2% not that much, but not null too

Reply:
I've changed that, happy to mark resolved and moved to test ready if that's ok for you both.